### ISSUE TITLE
Fails to build on F-Droid

### ISSUE THREAD

**Issue by jspricke** [x0b/rcx#131]
Hi,

the latest version (1.12.0-pre1) fails to build on F-Droid:
```
* Where:
Build file '/home/vagrant/build/io.github.x0b.rcx/rclone/build.gradle' line: 27

* What went wrong:
A problem occurred evaluating project ':rclone'.
> Could not find method ignoreExitValue() for arguments [true] on project ':rclone' of type org.gradle.api.Project.
```
https://monitor.f-droid.org/builds/log/io.github.x0b.rcx/180

Would be great if you could help us fix it.

**Comment by x0b** [comment]
Sorry for the trouble. For the previous version, we introduced quite a few hacks to get it to build it on F-Droid, which are no longer required and may now cause issues:

[io.github.x0b.rcx.yml#L57-61](https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/io.github.x0b.rcx.yml#L57-61)
```yml
      - sed -i "27i    environment 'GO111MODULE', 'off'" rclone/build.gradle
      - sed -i "27i    ignoreExitValue true" rclone/build.gradle
      - sed -i -e 's/commandLine '\''go'\''/commandLine '\''\/home\/vagrant\/build\/io.github.x0b.rcx\/golang\/go\/bin\/go'\''/g'
        rclone/build.gradle
      - sed -i -e 's/'\''ndk-bundle'\'',/'\''..\/android-ndk'\'', '\''r20b'\'',/g'
```

[The deletion of a few lines in the build script](https://github.com/x0b/rcx/compare/14987067e401c24402dd017e2a50d98597025375..870088eaf95e15e62f5cdc31639ec3ded2ad770f#diff-44fd27c2bcba1626aedb43a0ecad124b4adfbbef72646dcdb47a484c69854f5b) rclone/build.gradle has caused the other lines to shift, and the sed commands are no valid. 

However, I was aware of this and I did plan on submitting a MR to fdroid data. ~I'm not quite sure why F-Droid tries to build test release tags.~ [It seems we forgot to add a regex filter pattern to the UpdateCheckMode](https://f-droid.org/de/docs/Build_Metadata_Reference/#UpdateCheckMode).

Okay, here's what you can do now:
- remove version `1.12.0-pre1`
- add the regex filter `^v?[.0-9]*$` to the UpdateCheckMode ("boring" version numbers only)

Once the proper v1.12 is released (next week probably), we will also need to remove the first 2 lines of sed fixes. Hopefully that's all.

**Comment by jspricke** [comment]
Did that in https://gitlab.com/fdroid/fdroiddata/-/merge_requests/8842, thx!

**Comment by x0b** [comment]
Build fixed in [fdroid/fdroiddata#8934](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/8934).
